### PR TITLE
APIS-6736 - Play 3.0 upgrade

### PR DESCRIPTION
**Pull request by Tobias1087**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 
-addSbtPlugin("org.scoverage"      %  "sbt-scoverage"          % "2.0.9")
-addSbtPlugin("org.scalastyle"     %% "scalastyle-sbt-plugin"  % "1.0.0")
-addSbtPlugin("org.scalameta"      %  "sbt-scalafmt"           % "2.5.2")
-addSbtPlugin("ch.epfl.scala"      %  "sbt-bloop"              % "1.5.11")
-addSbtPlugin("ch.epfl.scala"      %%  "sbt-scalafix"          % "0.11.1")
+addSbtPlugin("org.scoverage"   % "sbt-scoverage"         % "2.0.9")
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
+addSbtPlugin("org.scalameta"   % "sbt-scalafmt"          % "2.5.2")
+addSbtPlugin("ch.epfl.scala"   % "sbt-bloop"             % "1.5.11")
```
Comment (AndySpaven) [inline project/plugins.sbt:15]: 1.5.13 probably

**Review by AndySpaven — APPROVED** [review]

**Comment by Tobias1087** [comment]
rebuild

File: project/plugins.sbt [hunk 2]
```diff
@@ -12,7 +12,6 @@ addSbtPlugin("com.typesafe.sbt"    % "sbt-digest"         % "1.1.4")
 addSbtPlugin("org.scoverage"   % "sbt-scoverage"         % "2.0.9")
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
 addSbtPlugin("org.scalameta"   % "sbt-scalafmt"          % "2.5.2")
-addSbtPlugin("ch.epfl.scala"   % "sbt-bloop"             % "1.5.11")
```
Comment (AndySpaven) [inline project/plugins.sbt]: Removing bloop might be an issue until we isolate it properly to devs own workspaces.

Comment (Tobias1087) [reply project/plugins.sbt]: Yea I added it back in.

**Review by AndySpaven — APPROVED** [review]

**Pull request merged**